src/main/java/com/kramekk1/medicalClinic/Doctor.java

```diff
@@ -29,7 +30,7 @@ public class Doctor {
             joinColumns = @JoinColumn(name = "doctor_id"),
             inverseJoinColumns = @JoinColumn(name = "institution_id")
     )
-    private List<Institution> institution;
+    private List<Institution> institutions = new ArrayList<>();
 
     public void update(UpdateDoctorCommand command) {
```

Comment:
public void update(UpdateDoctorCommand command, List<Institution> institutions) {
            this.email = command.getEmail();
        this.password = command.getPassword();
        this.firstname = command.getFirstname();
        this.surname = command.getSurname();
        this.specializationType = command.getSpecializationType();
   this.institutions = institutions;
}